src/components/settings/Settings.h

```diff
@@ -212,7 +212,7 @@ namespace Pinetime {
     private:
       Pinetime::Controllers::FS& fs;
 
-      static constexpr uint32_t settingsVersion = 0x0003;
+      static constexpr uint32_t settingsVersion = 0x0004;
```

Comment:
Oops, I accidentally forgot to undo the bump in settings version. I'll fix that in the next commit.